python/ray/dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -380,6 +380,58 @@ def max_plus_pending(max_resource, pending_resource):
             ),
         ],
     ),
+    Panel(
+        id=50,
+        title="Node TPU Tensorcore Utilization (Percentage)",
+        description="Percentage of tensorcore utilization for the TPUs on this node. Computed by dividing the number of tensorcore operations by the maximum supported number of operations during the sample period.",
+        unit="%",
+        targets=[
+            Target(
+                expr='sum(ray_tpu_tensorcore_utilization{{instance=~"$Instance",{global_filters}}}) by (instance, TpuIndex, TpuDeviceName)',
+                legend="Tensorcore Utilization: {{instance}}, tpu.{{TpuIndex}}, {{TpuType}}, {{TpuTopology}}",
+            ),
+        ],
+    ),
+    Panel(
+        id=51,
+        title="Node TPU High Bandwidth Memory Utilization (Percentage)",
+        description="Percentage of bandwidth memory utilization for the TPUs on this node. Computed by dividing the memory bandwidth used by the maximum supported memory bandwidth limit during the sample period.",
+        unit="%",
+        targets=[
+            Target(
+                expr='sum(ray_tpu_memory_bandwidth_utilization{{instance=~"$Instance",{global_filters}}}) by (instance, TpuIndex, TpuDeviceName)',
+                legend="Memory Bandwidth Utilization: {{instance}}, tpu.{{TpuIndex}}, {{TpuType}}, {{TpuTopology}}",
```

Comment:
Probably don't need `Memory Bandwidth Utilization:` in the legend since its the title of the graph itself.

Also for the other graphs as well.